### DISCUSSION TITLE
NaN output when solar is no longer producing

### DISCUSSION THREAD

**Discussion by DrSpaldo** [kalkih/mini-graph-card#859] [Q&A]
Hi, I was wondering what people did at the time when there is no longer any output in your graph? For me, it is when the solar inverter is no longer producing power, I still see the graph from the day, however, get the "NaN" output. Is there a way that I can make it no show any value once the output has stopped?

![image](https://user-images.githubusercontent.com/8526509/198563546-03cbdecd-53e3-49d5-8d99-358052244e3e.png)

**Comment by ildar170975** [comment]
> I still see the graph from the day

What else do you expect since you have "smoothing: true" & a small "points_per_hour"?
Also, that was just a speculation - no code was provided, no info about a source data (how often is changed, how often is polled, ...).

**Comment by DrSpaldo** [comment]
Seeing the graph from the day is great, I like that and I am not complaining about it. 

My question was more so about the "NaN" value and how can I make the current value disappear when it no longer has any output/reading? So the graph would just have nothing or 0 instead in that area..

Happy to show the code if it will help:

```
  - type: custom:mini-graph-card
    name: Solar
    entities:
      - sensor.power_ac_fronius_inverter_1_192_168_2_70
```

**Comment by ildar170975** [comment]
1. Your code is minimal. Means - using default values. Means - with smoothing and approximation. Means - wrong graphs. If it is ok for you - ok. 
2. If you want not to show NaN text - use card-mod to replace the text with nothing or something like “no data” text. There are no any other solutions so far. For card-mod solution - go to Community to the dedicated card-mod thread, check the 1st post -> link at the bottom -> styles for mini-graph-card.

**Comment by DrSpaldo** [comment]
For those that find this thread and don't want to spend hours looking for a needle in haystack, here are a couple of links with examples:

https://community.home-assistant.io/t/lovelace-mini-graph-card/71385/1705

https://community.home-assistant.io/t/lovelace-mini-graph-card/71385/1585
